staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/validator.go

```diff
@@ -28,17 +28,15 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/util/validation/field"
-	"k8s.io/kube-openapi/pkg/validation/validate"
 
 	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
-	apiservervalidation "k8s.io/apiextensions-apiserver/pkg/apiserver/validation"
+	apiextensionsvalidation "k8s.io/apiextensions-apiserver/pkg/apiserver/validation"
 )
 
 type customResourceValidator struct {
-	namespaceScoped       bool
-	kind                  schema.GroupVersionKind
-	schemaValidator       *validate.SchemaValidator
-	statusSchemaValidator *validate.SchemaValidator
```

Comment:
why is this changed? At the very least it doubles the runtime of validation because we always validate spec and status. Is there any need for this change to get ratcheting working?

Reply:
This removed the statusSchemaValidator. I removed it because I was replacing all the usages of SchemaValidator with a RatchetingSchemaValidator. I saw it was unused, so pulled the entire thread out. I'm not sure how doing this doubles runtime since it was unused.

Reply:
Oh, isn't that just a bug of the old code?

See:
```
allErrs = append(allErrs, apiservervalidation.ValidateCustomResource(nil, u.UnstructuredContent(), a.schemaValidator)...)
```
in `ValidateStatusUpdate`.

It's functionally correct, but there is no need to do validation and all the complexity of things outside of status when updating only the status.